dist/APIManager.js

```diff
@@ -0,0 +1,52 @@
+class APIManager {
+  constructor(){
+    this.players = []
+    this.roster = []
+    this.dream = []
+  }
+  getPlayers(teamName, renderCallback){
```

Comment:
APIManager should just handle the data. Therefore, it shouldn't get and/or invoke any rendering functions - those should be invoked in the main.js file.